### ISSUE TITLE
tracking information 

### ISSUE THREAD

**Issue by yaira24** [Jack-CV/FaceKit#77]
Hello, @Jack-CV thanks for your excellent job! 
Can I get more information on the tracking:
# what is the tracking algorithm?
# where is the check that the current detection is detection that has a tracking "obj" before 
# how can we add labels to see that the tracking is working

I will like to show id or same kind of label on the detection to see how the tracking works 

thanks

**Comment by bmabir17** [comment]
@yaira24  were you able to add id info to tracked face? It seems that tracking is only going on for 30 frames by default. And can be change with tracking_period variable while initializing the detector. Dont think this tracker is keep the track of the face until it disappears or assigns a unique id to each face.

**Comment by yaira24** [comment]
no,  I was not able to do this. its track the box, not the face
